xmake/core/main.lua

```diff
@@ -74,6 +85,11 @@ function main._help()
             utils.cprint(menu.copyright)
         end
 
+        -- print features
+        if menu.features then
```

Comment:
Right, It might be better to use `xmake --version [-v|--verbose]`. Only developers will care about verbose info.